rfcs/0007_reorganize_repositories.md

```diff
@@ -0,0 +1,74 @@
+- Start Date: 2025-01-28
+- RFC PR: https://github.com/thousandbrainsproject/tbp.monty/pull/159/
+
+# Summary
+
+The proposal is to establish a better way of organizing Thousand Brains Project repositories.
+
+Currently there are two repositories:
+
+1. `tbp.monty` - This is the main repository for the Monty framework.
+2. `monty_lab` - This is what `nupic.monty/projects` used to be, and is currently a catch-all for code that is not part of the main Monty framework.
+
+The high-level proposal is to break-up `monty_lab` where appropriate, and have a lower threshold for creating new, independent repositories.
+
+More specifically, the proposed structure and description of each repository would be:
+
+`tbp.monty`
+- Core code-base for the Monty framework.
+- All code has undergone thorough review, and Circle-CI runs unit-tests and style checks on any new code.
+- Should not become bloated with experimental code that is unlikely to be re-used in future work.
```

Comment:
I would make it even more explicit. Something like "should not contain experimental code. The only experiment configs in this repository are those used for our benchmark experiments"

Reply:
Thanks, I've split this up and clarified it a bit more.